benchmarks/index.js

```diff
@@ -0,0 +1,33 @@
+var DataSource = require('loopback-datasource-juggler').DataSource;
+var connector = require('..');
+var Benchmark = require('benchmark');
+
+var ds = new DataSource(connector, {
+  host: process.env.LB_HOST || '127.0.0.1',
+  port: process.env.LB_PORT || 27017,
+  database: process.env.LB_DB || 'strongloop'
+});
+var Todo = ds.define('Todo', {
+  content: {type: String}
+});
+
+var suite = new Benchmark.Suite;
+suite
+  .on('start', function() {
+    console.log('#', new Date());
+  })
+  .add('Todo.create', function() {
+    Todo.create({content: 'Buy eggs'});
+  })
+  .add('Todo.find', function() {
+    Todo.find();
+  })
+  .on('cycle', function(event) {
+    console.log(String(event.target));
+  })
+  .on('complete', function(event) {
+    console.log();
+    Todo.destroyAll();
+    process.exit();
```

Comment:
@raymondfeng The node app stays open when done. I think this is because the datasource still has a connection open to the database. Is there a way to force a disconnect manually (ie. `ds.connector.disconnect`)?